### PR TITLE
lightning: add doc for send-kv-size

### PR DESCRIPTION
**Pull request by D3Hunter**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -134,8 +134,13 @@ addr = "172.16.31.10:8287"
 #  - none: 不检测重复记录。该模式是三种模式中性能最佳的，但是如果数据源存在重复记录，会导致 TiDB 中出现数据不一致的情况。
 #  - remove: 记录所有目标表中的重复记录，和 'record' 模式相似。但是会删除目标表所有的重复记录，以确保目标 TiDB 中的数据状态保持一致。
 # duplicate-resolution = 'none'
-# 物理导入模式一次请求中发送的 KV 数量。
+# 物理导入模式下，向 TiKV 发送数据时一次请求中最大 KV 数量。
+# 自 v7.2.0 开始，该参数 deprecated，设置后不在生效，如果希望调整一次请求中向 TiKV 发送的数据量，请使用 send-kv-size 参数
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:138]:
```suggestion
# 自 v7.2.0 开始，该参数 deprecated，设置后不再生效。如果希望调整一次请求中向 TiKV 发送的数据量，请使用 `send-kv-size` 参数。
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 2]
```diff
@@ -134,8 +134,13 @@ addr = "172.16.31.10:8287"
...
 # send-kv-pairs = 32768
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:139]:
```suggestion
# send-kv-pairs = 3200
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 3]
```diff
@@ -134,8 +134,13 @@ addr = "172.16.31.10:8287"
...
+# 物理导入模式下，向 TiKV 发送数据时一次请求的最大大小。
+# 默认值为 16K，一般导入时不建议调整该参数。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:141]:
```suggestion
# 默认值为 "16K"，一般情况下不建议调整该参数。
```

File: tidb-lightning/tidb-lightning-configuration.md (hunk 2 above)
Comment (D3Hunter) [reply tidb-lightning/tidb-lightning-configuration.md:139]: 这块没改

**Review by hfxsd — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 4]
```diff
@@ -134,8 +134,13 @@ addr = "172.16.31.10:8287"
 #  - none: 不检测重复记录。该模式是三种模式中性能最佳的，但是如果数据源存在重复记录，会导致 TiDB 中出现数据不一致的情况。
 #  - remove: 记录所有目标表中的重复记录，和 'record' 模式相似。但是会删除目标表所有的重复记录，以确保目标 TiDB 中的数据状态保持一致。
 # duplicate-resolution = 'none'
-# 物理导入模式一次请求中发送的 KV 数量。
-# send-kv-pairs = 32768
+# 物理导入模式下，向 TiKV 发送数据时一次请求中最大 KV 数量。
+# 自 v7.2.0 开始，该参数废弃，设置后不再生效。如果希望调整一次请求中向 TiKV 发送的数据量，请使用 `send-kv-size` 参数。
+# send-kv-pairs = 3200
```
Comment (Frank945946) [inline tidb-lightning/tidb-lightning-configuration.md:139]:
```suggestion
```

Comment (Frank945946) [reply tidb-lightning/tidb-lightning-configuration.md:139]: @D3Hunter 这行是不是得删除掉

Comment (hfxsd) [reply tidb-lightning/tidb-lightning-configuration.md:139]: 已经在这个 PR 里改成 3200 了 https://github.com/pingcap/docs-cn/pull/13993/files#diff-e60eb511e8e7fcd107a7d3a7d3da20c1e422686d723c4f14446d0c05cb22c08b
最佳实践里也写的是 3200 https://pingcap.feishu.cn/wiki/YYaTwiTEdiaF88kiL9ycfVQsnrf

Comment (hfxsd) [reply tidb-lightning/tidb-lightning-configuration.md:139]: 经过讨论，先保留 send-kv-pairs = 32768

**Review by Frank945946 — APPROVED** [review]
LGTM

File: tidb-lightning/tidb-lightning-configuration.md [hunk 5]
```diff
@@ -134,8 +134,13 @@ addr = "172.16.31.10:8287"
 #  - none: 不检测重复记录。该模式是三种模式中性能最佳的，但是如果数据源存在重复记录，会导致 TiDB 中出现数据不一致的情况。
 #  - remove: 记录所有目标表中的重复记录，和 'record' 模式相似。但是会删除目标表所有的重复记录，以确保目标 TiDB 中的数据状态保持一致。
 # duplicate-resolution = 'none'
-# 物理导入模式一次请求中发送的 KV 数量。
+# 物理导入模式下，向 TiKV 发送数据时一次请求中最大 KV 数量。
+# 自 v7.2.0 开始，该参数废弃，设置后不再生效。如果希望调整一次请求中向 TiKV 发送的数据量，请使用 `send-kv-size` 参数。
 # send-kv-pairs = 32768
+# 物理导入模式下，向 TiKV 发送数据时一次请求的最大大小。
+# 默认值为 "16K"，一般情况下不建议调整该参数。
```
Comment (ran-huang) [inline tidb-lightning/tidb-lightning-configuration.md:141]: 16K 的意思是 16KB 吗？

Comment (D3Hunter) [reply tidb-lightning/tidb-lightning-configuration.md:141]: 是的

**Review by ran-huang — APPROVED** [review]

**Pull request merged**